docs/application/native/guides/applications/efl-ui-app.md

```diff
@@ -225,6 +225,10 @@ Since Tizen 2.4, an application is not allowed to run in the background except w
 | Sensor (context)               | Processing context data from the sensors, such as gesture | Sensor API (in [mobile](../../api/mobile/latest/group__CAPI__SYSTEM__SENSOR__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__SYSTEM__SENSOR__MODULE.html) applications) | `sensor`                                 |
 | IoT Communication/Connectivity | Communicating between external devices in the background (such as Wi-Fi and Bluetooth) | Wi-Fi (in [mobile](../../api/mobile/latest/group__CAPI__NETWORK__WIFI__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__NETWORK__WIFI__MODULE.html) applications) and Bluetooth API (in [mobile](../../api/mobile/latest/group__CAPI__NETWORK__BLUETOOTH__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__NETWORK__BLUETOOTH__MODULE.html) applications) | `iot-communication`                      |
 
+  > **Note**
```

Comment:
Suggest rephrasing to:

Since Tizen 4.0, even if the background network category is declared, the running application stops if the network is not connected.

The reason I replaced freeze with stop is: "Freeze" usually is used to refer a code freeze.  Therefore, in context with application, stops is more appropriate.

Reply:
done

Reply:
@hjhun,

Please remove the hyphen between "background-network "

In the table there is no hyphen. Please change to:

Since Tizen 4.0, even if the background network category is declared, the running application stops if the network is not connected.

Reply:
done.